configure.ac

```diff
@@ -202,6 +204,114 @@ AS_CASE([$with_gc],
     )
 AC_MSG_RESULT([$with_gc])
 
+dnl
+dnl User setting: native thread-local storage (off by default)
```

Comment:
Not saying it should be different, but just to understand better: why is this off by default? What's the pro/con for turning it on (or off) by default?

Reply:
Performance and portability. The stack-based TLS approach is the fastest *and* the most portable one, but it cannot be used with arbitrary thread stacks (i.e. when using it GAP as a library). One could make a good argument that it should be the other way round; in either case, there should be an explanation.